youtube_dl/extractor/neteasemusic.py

```diff
@@ -84,7 +84,8 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'id': '32102397',
             'ext': 'mp3',
             'title': 'Bad Blood (feat. Kendrick Lamar)',
-            'creator': 'Taylor Swift / Kendrick Lamar',
```

Comment:
`creator` should not necessarily contain artist in terms of a person or band. It can be a studio/channel/whatever. The  meaning of `creator` has expanded from the initial description wording.